demo/src/screens/componentScreens/ScrollBarScreen.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import {View, Text, ScrollBar, Button} from 'react-native-ui-lib';
+class ScrollBarScreen extends Component {
+  state = {
+    selectedIndex: 0
```

Comment:
'focusIndex' doesn't work if I set the initial value to 6, for example, since you only apply it in ScrollBar's componentDidUpdate()

Reply:
you mean it doesn't work as an initial value, only changing value?

Reply:
Yes

Reply:
fixed